app/src/main/java/com/example/home/stream/StreamFragment.java

```diff
@@ -0,0 +1,133 @@
+package com.example.home.stream;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+
+import com.codepath.asynchttpclient.AsyncHttpClient;
+import com.codepath.asynchttpclient.RequestHeaders;
+import com.codepath.asynchttpclient.RequestParams;
+import com.codepath.asynchttpclient.callback.JsonHttpResponseHandler;
+import com.example.home.R;
+import com.example.home.models.Home;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import okhttp3.Headers;
+
+
+public class StreamFragment extends Fragment {
+
+    public String mNoOfBedrooms;
+    public String mPropertyTypeText;
+    public Integer mZipCode;
+    public List<Home> mHome;
+    public HomeAdapter adapter;
+    public RecyclerView streamRecyclerView;
+    public LinearLayout StreamLinearLayout;
+    public static final String LATITUDE_PARAM = "latitude";
+    public static final String LONGITUDE_PARAM = "longitude";
+    public static final String RADIUS_PARAM = "radius";
+    public static final String MIN_BEDS_PARAM = "minbeds";
+    public static final String MAX_BEDS_PARAM = "maxbeds";
+    public static final String PROPERTY_TYPE_PARAM = "propertytype";
+
+
+    public StreamFragment(String mNoOfBedrooms, String mPropertyTypeText, Integer mZipCode) {
```

Comment:
you don't need to m Prefix here since those are local variables. Also, I would suggest receiving a User Object from which then you later extract UserPreferences to know which houses to render